doc/rdesktop.1

```diff
@@ -123,6 +128,11 @@ but everything after this is unencrypted (including interactive logins).
 Do not send mouse motion events.  This saves bandwidth, although some Windows
 applications may rely on receiving mouse motion.
 .TP
+.BR "-M"
```

Comment:
Yeah it was. Annoyingly the mouse cursor was only broke about half of the time (I had seen this on 2012R2 and had to double remote to fix it). I think the problem actually got fixed in trunk after I submitted all of this, but the fix won't work right if -m is used.

I couldn't use any of the fixes until they were all in though. The Win10 login screen is affected by the mouse problem.